scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -132,6 +132,18 @@ class FossId internal constructor(
         @JvmStatic
         internal val PROJECT_REVISION_LABEL = "projectVcsRevision"
```

Comment:
commit-msg: Please explain `why` the added data is needed. E.g. what are the use cases?